deploy/crds/managed.openshift.io_customdomains.yaml

```diff
@@ -49,11 +49,11 @@ spec:
                 description: Certificate points to the custom TLS secret
                 properties:
                   name:
-                    description: Name is unique within a namespace to reference a
+                    description: name is unique within a namespace to reference a
```

Comment:
Why was this changed?

Reply:
To be honest I do not know. This is something that was breaking on the prow validate and  @sam-nguyen7 privided me with this command `./boilerplate/_lib/container-make generate ` which made those changes as Fix/Workaround . @sam-nguyen7 if you can give more context on this please do so.

Reply:
That's odd, there might be a bug with that make target then.